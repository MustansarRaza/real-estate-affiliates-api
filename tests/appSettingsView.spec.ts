import { loadSequelizeFixtures, useRequestFactory } from "./support";
import { ModelFixtureNames } from "./support/data";

describe("GET /markets/secondary/listings", () => {
  beforeEach(async () => {
    await loadSequelizeFixtures(ModelFixtureNames.APP_RELEASE);
  });

  it("returns current and minimum app version for affiliates app", async () => {
    const { body } = await useRequestFactory()
      .withVerifiedUser()
      .get(`/appRelease`)
      .expect(200);

    expect(body).toMatchSnapshot();
  });
});
