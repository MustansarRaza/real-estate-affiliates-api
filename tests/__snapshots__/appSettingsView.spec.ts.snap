// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /markets/secondary/listings returns current and minimum app version for affiliates app 1`] = `
Object {
  "appVariables": Object {
    "min_version": "0.1.0",
  },
  "version": "0.1.1",
}
`;
